lib/service/signals.go

```diff
@@ -134,6 +133,37 @@ func (process *TeleportProcess) WaitForSignals(ctx context.Context) error {
 	}
 }
 
+func (process *TeleportProcess) writeToSignalPipe(message string) error {
+	signalPipe, err := process.importSignalPipe()
+	if err != nil {
+		if !trace.IsNotFound(err) {
+			return trace.Wrap(err)
+		}
+		log.Debugf("No signal pipe found, not going to signal")
+		return nil
+	}
+	defer signalPipe.Close()
+
+	messageSent, cancel := context.WithCancel(context.TODO())
```

Comment:
Since in this case `context.TODO()` will never be replaced (it's only used to signal when the write is complete), I'd use `context.Background()`. That way it won't highlight `TODO` in editors.